### PR TITLE
Ianhelle/1.5.0post fixes

### PR DESCRIPTION
**Pull request by ianhelle**
Post fixes for 1.5.0pre1
Graph and MDE providers
Kusto driver

### CODE REVIEW

File: docs/source/data_acquisition/DataProviders.rst [hunk 1]
```diff
@@ -386,25 +386,36 @@ Connecting to an OData Source
 -----------------------------
 :py:mod:`OData driver API documentation<msticpy.data.drivers.odata_driver>`
 
-You can also connect to OData based data sources such as the MDATP API,
-or the Security Graph API. These connections often rely on having a
-dedicated Azure AD app for handling the authentication process.
+You can also connect to OData based data sources such as the Microsoft Defender,
```
Comment (petebryan) [inline docs/source/data_acquisition/DataProviders.rst:389]: Did you mean Microsoft 365 Defender here? I think Microsoft Defender is the proper name for Defender AV

File: docs/source/data_acquisition/DataProviders.rst [hunk 2]
```diff
@@ -386,25 +386,36 @@ Connecting to an OData Source
...
+Microsoft Defender for Endpoint, and the Microsoft Graph APIs.
+These connections often rely on having a
+dedicated Azure AD app (plus app secret) for handling the authentication process.
 
-MDATP
-~~~~~
+Microsoft Defender
```
Comment (petebryan) [inline docs/source/data_acquisition/DataProviders.rst:394]: I think we should use Microsoft 365 Defender here

File: msticpy/data/queries/kql_mdatp_alerts.yaml [hunk 3]
```diff
@@ -1,39 +1,39 @@
 metadata:
   version: 1
   description: MDATP Queries
-  data_environments: [MDATP]
+  data_environments: [MDATP, MDE, MD365]
```
Comment (petebryan) [inline msticpy/data/queries/kql_mdatp_alerts.yaml:4]: Given the product is Microsoft 365 Defender so we want to make this M365D?

**Review by petebryan — APPROVED** [review]

File: msticpy/data/queries/kql_mdatp_alerts.yaml (hunk 3 above)
Comment (ianhelle) [reply msticpy/data/queries/kql_mdatp_alerts.yaml]: Yep - changed all to M365D

**Comment by ianhelle** [comment]
Fixed issues (and a couple of test failures) identified by @petebryan

**Pull request merged**